integrations/osmosis/packages/abstract-adapter/src/staking.rs

```diff
@@ -235,21 +211,23 @@ pub mod fns {
 
         fn query_staked(
             &self,
-            querier: &QuerierWrapper,
-            staker: Addr,
+            _querier: &QuerierWrapper,
+            _staker: Addr,
             _unbonding_period: Option<cw_utils::Duration>,
         ) -> Result<StakeResponse, CwStakingError> {
-            // We query all the locked tokens that correspond to the token in question
-            let lockup_request = LockupQuerier::new(querier);
-            let locked_up = lockup_request
-                .account_locked_coins(staker.to_string())?
-                .coins
-                .into_iter()
-                .filter(|coin| coin.denom == self.lp_token.clone().unwrap())
-                .map(|lock| Uint128::from_str(&lock.amount).unwrap())
-                .sum();
-
-            Ok(StakeResponse { amount: locked_up })
+            Err(CwStakingError::NotImplemented("osmosis".to_owned()))
+            // TODO: whitelist for contracts
+            // let lockup_request = LockupQuerier::new(querier);
+            // let locked_up = lockup_request.account_locked_coins(staker.to_string())?;
```

Comment:
Wonder what should we do to this method after they allow the message. It's still not efficient in case user decides to look up for multiple LPs in a single TX. Should we create another method(or change this one) to query multiple LPs?